Git Formats/Mailmap.sublime-syntax

```diff
@@ -0,0 +1,18 @@
+%YAML 1.2
+---
+# Syntax based on documentation here:
+# https://www.git-scm.com/docs/git-check-mailmap#_mapping_authors
+
+name: Mailmap
+file_extensions:
+  - .mailmap
+  - mailmap
+scope: text.git.mailmap
+
+contexts:
+  main:
+    - include: Git Common.sublime-syntax#comments
+    - include: Git Common.sublime-syntax#email
+    - match: '[ \t]*(\b[^<#\n]+?)[ \t]*(?=<|$)'
```

Comment:
Matching for `[ \t]*` at the start is *probably* redundant? Idk how greedy the other includes are.

Reply:
You are right. I had that in there before adding the `\b`. I'll take it out.

I am _hoping_ that the `\b` is okay to have here. I haven't looked at valid *Nix and Windows user names. Of course there may be some goofball who decides they want to be known as `%le3t` or something. -_-;

Reply:
Fixed it for goofballs, too. 😄